lib/utils/users.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash';
```

Comment:
Remember to import only the methods you want to use from `lodash` instead of the whole module
```suggestion
import _get from 'lodash/get';
```